lms/djangoapps/courseware/masquerade.py

```diff
@@ -106,6 +146,14 @@ def is_masquerading_as_student(user, course_key):
     return get_masquerade_role(user, course_key) == 'student'
 
 
+def is_masquerading_as_specific_student(user, course_key):  # pylint: disable=invalid-name
+    """
+    Returns whether the user is a staff member masquerading as a specific student.
+    """
+    course_masquerade = get_course_masquerade(user, course_key)
+    return bool(course_masquerade and course_masquerade.user_name)
```

Comment:
Nit: maybe `CourseMasquerade.__nonzero__` should just check `course_masquerade.user_name`